hash/n64.xml

```diff
@@ -6872,7 +6872,19 @@ This cart features a RTC, currently unemulated
 	</software>
 
 <!--
-This cart features two SmartMedia slots to import pictures to manipulate in the game, currently unemulated
+This cart features two SmartMedia slots to import pictures to manipulate in the game, currently unemulated.
+Known SmartMedia cards:
+Dumped Title                                                Japanese title                          Notes
+  YES  Illust-shuu - Postcard 1                             イラムト集 ポストカード 1
+  YES  Illust-shuu - Omoshiro Accessory 1                   イラムト集 おもしろアクセサリー 1
```

Comment:
You still have “Illust“ misspelled as the nonsensical “Iramuto” in the Japanese titles here.

Reply:
Fixed, thanks for your patiente